filebeat/config.sls

```diff
@@ -1,32 +1,13 @@
-{% from "filebeat/map.jinja" import conf with context %}
+# -*- coding: utf-8 -*-
+# vim: ft=sls
 
-{% if salt['pillar.get']('filebeat:logstash:tls:enabled', False)  %}
-{{ salt['pillar.get']('filebeat:logstash:tls:ssl_cert_path', '/etc/pki/tls/certs/logstash-forwarder.crt') }}:
-  file.managed:
-    - source: {{ salt['pillar.get']('filebeat:logstash:tls:ssl_cert', 'salt://filebeat/files/ca.pem') }}
-    - template: jinja
-    - makedirs: True
-    - user: root
-    - group: root
-    - mode: 644
-    - watch_in:
-      - filebeat.config
-{% endif %}
```

Comment:
this expects users to maintain their own tls config for filebeat seperately. this is a loss of relevant functionality, and will break existing configs.